kornia/testing/__init__.py

```diff
@@ -193,3 +193,19 @@ def KORNIA_CHECK(condition, msg: Optional[str] = None):
 def KORNIA_CHECK_IS_TENSOR(x, msg: Optional[str] = None):
     if not isinstance(x, Tensor):
         raise TypeError(f"Not a Tensor type. Got: {type(x)}.\n{msg}")
+
+
+def KORNIA_CHECK_IS_RGB(x: Tensor, msg: Optional[str] = None):
+    if len(x.shape) < 3 or x.shape[-3] != 3:
+        raise TypeError(f"Not an RGB tensor. Got: {type(x)}.\n{msg}")
+
+
+def KORNIA_CHECK_IS_GRAY(x: Tensor, msg: Optional[str] = None):
+    if len(x.shape) < 3 or x.shape[-3] != 1:
+        raise TypeError(f"Not a gray tensor. Got: {type(x)}.\n{msg}")
+
+
+def KORNIA_CHECK_IS_RGB_OR_GRAY(x: Tensor, msg: Optional[str] = None):
```

Comment:
```suggestion
def KORNIA_CHECK_IS_COLOR_OR_GRAY(x: Tensor, msg: Optional[str] = None):
```
Let’s call it color, the channels ordering is another issue to address which I was trying here (pr is blocked i found limitations again with torchscript)

https://github.com/kornia/kornia/pull/1562/files#diff-83556457406cc619f39c836c4cb739c29e6f27971e35c97683edc9287bf21038R71